007_team_5_agent_design/007_team_5_agent_design.tex

```diff
@@ -1 +1,522 @@
+
+% Team 5 also performs well with Team 7 and Team 3; these results are shown in Tables \ref{tab:team5-team5-team7} and \ref{tab:team5-team5-team3}, respectively.
+
+% \begin{table}
+%     \centering
+%     \begin{tabular}{|c|c|c|c|}
+%         \hline
+%         \textbf{Food per agent ratio} & \textbf{6} & \textbf{7} & \textbf{8}\\
+%         \hline
+%         Team 5 deaths and average age upon death       & 4, 46 & 0, 0 & 0, 0\\
+%         \hline
+%         Team 7 deaths and average age upon death       & 4, 19.5 & 3, 25 & 1, 22\\
+%         \hline
+%     \end{tabular}
+%     \caption{Results of a Tower consisting of Team 5 and Team 7 agents}
+%     \label{tab:team5-team5-team7}
+% \end{table}
+
+% \begin{table}
+%     \centering
+%     \begin{tabular}{|c|c|c|c|}
+%         \hline
+%         \textbf{Food per agent ratio} & \textbf{6} & \textbf{7} & \textbf{8}\\
+%         \hline
+%         Team 5 deaths and average age upon death       & 11, 41 & 7, 63 & 1, 15\\
+%         \hline
+%         Team 3 deaths and average age upon death       & 8, 45 & 0, 0 & 0, 0 \\
+%         \hline
+%     \end{tabular}
+%     \caption{Results of a Tower consisting of Team 5 and Team 3 agents}
+%     \label{tab:team5-team5-team3}
+% \end{table}
+
+% \subsubsection*{Foes}
+% Team 5 does not perform well with either Team 6 or Team 2. When paired with Team 6, the Team 6 agents die often and have less than half the lifespan of the Team 5 agents. This issue persists with increased amount of food on the platform.
+
+% When paired with Team 2, the Team 2 agent has significantly fewer deaths than the Team 5 agent and also has a longer lifespan. These results suggest that this agent pairing is unable to self-organise, with the potentially more selfless Team 5 agents left to starve by the Team 2 agents. This is because the Team 2 agents do not communicate except to ask for the HP of other agents.
+
+% The Team 5 agent is least compatible with random agents (i.e. agents which take a random amount of food). The total number of deaths is highest and average age upon death is lowest with this pairing. The life expectancy also does not improve with more food on the platform.
+
+\subsection*{Self-Organisation With Other Agents}
+The overall aim within the tower is for agents to self-organise and create a stable society. Tests were run to investigate how compatible the Team 5 agent is with each of the agents designed by each of the other teams. Simulations were run with a tower consisting of Team 5 agents and one other agent type, with seven agents each and five food available per agent, to see whether they could self-organise within 100 days. These results are shown in Table \ref{tab:team5-self-org-agents}
```

Comment:
uncut the above commented section? or if this is for space then at least another paragraph of analysis needed here

Reply:
I think the uncommented bit says the same information in a more concise manner imo